src/main/scala/net/nmoncho/sbt/dependencycheck/tasks/package.scala

```diff
@@ -59,6 +60,7 @@ package object tasks {
       Thread.currentThread().setContextClassLoader(newClassLoader)
 
       try {
+        Downloader.getInstance().configure(settings)
```

Comment:
Naive question, because I don't know SBT well enough:
Is there only one task ever running? 

My assumptions: When I call `sbt dependencyCheck` I thing multiple dependencyCheck task can run in a multi-project setup. Each project could have different settings, then we 'stomp' over the global settings, creating extrem hard to debug issues.

So, my naive reading then:
- Maybe there is a SBT mechanism to avoid it.
- Otherwise: I would synchronize this block =(. Its ugly, but I prefer it over having super weird concurrency issues.

Reply:
Hi @gamlerhart,
Tasks can run in parallel. This commit should take care of any race conditions: https://github.com/nMoncho/sbt-dependency-check/commit/36e163d7060d353c9d10031b74619e111cf289e2 (not released yet)

And to your point, yes, configuration can become extremely complicated if not used appropriately. My hope is that configuration is set once project-wide, even on a multi-project setup, to avoid any headaches and second guesses.